src/queries/text.ts

```diff
@@ -54,7 +54,7 @@ const getChildrenAsText = (children: React.ReactChild[]) => {
   return textContent;
 };
 
-const getNodeByText = (
+export const getNodeByText = (
```

Comment:
As it is used in two different queries, I thought about moving it to `helpers` folder.

Reply:
`helpers/matchers/text.ts` should be good

Reply:
Hey, I've moved it to `helpers/matchers/matchTextContent`. Is it okay?